src/main/scala/com/fulcrumgenomics/umi/CallMolecularConsensusReads.scala

```diff
@@ -69,6 +69,13 @@ import htsjdk.samtools._
     |calls each end of a pair independently, and does not jointly call bases that overlap within a pair.  Insertion or
     |deletion errors in the reads are not considered in the consensus model.
     |
+    |Particular attention should be paid to setting the --min-reads parameter as this can have a dramatic effect on
+    |both results and runtime.  For libraries with low duplication rates (e.g. 100-300X exomes libraries) in which it
+    |is desirable to retain singleton reads while making consensus reads from sets of duplicates, --min-reads=1 is
+    |appropriate.  For libraries with high duplication rates where it is desirable to only produce consensus reads
+    |supported by 2+ reads to allow error correction, --min-reads=2 or higher is appropriate.  After generation,
+    |consensus reads can be further filtered using the FilterConsensusReads tool.
```

Comment:
Also mention that the main reason to have `--min-reads` is performance, namely to upfront shrink the data size and not have a big BAM.